### PR TITLE
EOS-15343 : S3 IEM Alerts correction/improvement

### PR DESCRIPTION
**Pull request by pranavPawar07**
Signed-off-by: Pranav <pranav.pawar@seagate.com>

### CODE REVIEW

**Review by bkirunge7 — COMMENTED** [review]
Pls check background service alerts as well

**Review by bkirunge7 — APPROVED** [review]

File: server/s3_iem.h [hunk 1]
```diff
@@ -66,37 +66,49 @@ extern S3Option* g_option_instance;
...
 #define S3_IEM_CHUNK_PARSING_FAIL "0030020001"
-#define S3_IEM_CHUNK_PARSING_FAIL_STR "Chunk parsing failed"
+#define S3_IEM_CHUNK_PARSING_FAIL_STR \
+  "Invalid chunk payload. For more information, refer troubleshooting guide."
 #define S3_IEM_CHUNK_PARSING_FAIL_JSON ""
 
 #define S3_IEM_MOTR_CONN_FAIL "0030030001"
-#define S3_IEM_MOTR_CONN_FAIL_STR "Motr connection failed"
+#define S3_IEM_MOTR_CONN_FAIL_STR                                          \
+  "Clovis connection failed. For more information, refer troubleshooting " \
```
Comment (sachitanands) [inline server/s3_iem.h:81]: please remove Clovis word and use Motr client may be ?

**Review by shalakadharap — APPROVED** [review]
Kindly address @sachitanands comment.
Rest all looking good.

File: auth/server/src/main/java/com/seagates3/util/BinaryUtil.java [hunk 2]
```diff
@@ -239,8 +235,7 @@ public static String base64EncodedHash(String text) {
 
             return md.digest();
         } catch (NoSuchAlgorithmException ex) {
-            IEMUtil.log(IEMUtil.Level.ERROR, IEMUtil.SHA256_UNAVAILABLE,
-                    "Algorithm SHA-256 not available", null);
+            LOGGER.error"Algorithm SHA-256 not available.");
```
Comment (sachitanands) [inline auth/server/src/main/java/com/seagates3/util/BinaryUtil.java:238]: starting bracket is missing ..pls add it.
After all these fixes please try to build auth server  as ,
cd cortx-s3server/auth/
./mvnbuild.sh clean
./mvnbuild.sh package

File: server/s3_iem.h (hunk 1 above)
Comment (pranavPawar07) [reply server/s3_iem.h]: Renamed clovis to motr.

File: auth/server/src/main/java/com/seagates3/util/BinaryUtil.java (hunk 2 above)
Comment (pranavPawar07) [reply auth/server/src/main/java/com/seagates3/util/BinaryUtil.java]: added '(' which was missing.

File: server/s3_iem.h (hunk 1 above)
Comment (mandarjoshi-techwriter) [reply server/s3_iem.h]: It should be "... refer Troubleshooting Guide."
Add full stop (.) after every error message.

**Comment by pranavPawar07** [comment]
PR : http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/213/

File: auth/server/src/main/java/com/seagates3/authentication/AWSV2Sign.java [hunk 3]
```diff
@@ -62,8 +62,7 @@ public Boolean authenticate(ClientRequestToken clientRequestToken,
                     requestor.getAccesskey().getSecretKey().getBytes(),
                     stringToSign.getBytes("UTF-8"));
         } catch (UnsupportedEncodingException ex) {
-            IEMUtil.log(IEMUtil.Level.ERROR, IEMUtil.UTF8_UNAVAILABLE,
-                    "UTF-8 encoding is not supported", null);
+          LOGGER.error("UTF-8 encoding is not supported.");
```
Comment (ujjwalpl) [inline auth/server/src/main/java/com/seagates3/authentication/AWSV2Sign.java:65]: Please fix indentation

**Pull request merged**